### ISSUE TITLE
breaktime not working

### ISSUE THREAD

**Issue by tuntor689** [Auros/Enhancements#11]
here's the [json file](https://drive.google.com/file/d/1zvBD7W-DWxyBVDLMB-VGKhiUmo5TGJIr/view?usp=sharing) but no logs (cuz i don't know where to find them), everything else works just not the breaktime

**Comment by Auros** [comment]
I'm gonna need a little more than "just doesn't work". Could you provide a log and also send me your Enhancements.json config file?

**Comment by tuntor689** [comment]
> I'm gonna need a little more than "just doesn't work". Could you provide a log and also send me your Enhancements.json config file?

oh yeah sorry i was tired, i couldn't find a log but i could find the e[nhancements.json](https://drive.google.com/file/d/1zvBD7W-DWxyBVDLMB-VGKhiUmo5TGJIr/view?usp=sharing) file

**Comment by Auros** [comment]
Logs are located in `Beat Saber/Logs`. Also, the url you linked is restricted. Could you make it public?

**Comment by tuntor689** [comment]
> Logs are located in `Beat Saber/Logs`. Also, the url you linked is restricted. Could you make it public?

Yeah I couldn't find a enhancements log, unless you were looking for the normal game log? Also, it should be public now, thanks.

**Comment by Auros** [comment]
A normal game log will suffice.
